bootstrap/secret/secret.go

```diff
@@ -43,6 +45,8 @@ const (
 	secretsStoredMetricName           = "SecuritySecretsStored"
 	securityConsulTokensRequestedName = "SecurityConsulTokensRequested"
 	securityConsulTokenDurationName   = "SecurityConsulTokenDuration"
+	securitySecretTokenDurationName   = "SecuritySecretTokenDuration"
```

Comment:
If this is for runtime token only then the name should be explicit.
```suggestion
	securityRuntimeSecretTokenDurationName   = "SecurityRuntimeSecretTokenDuration"
```